code/modules/security levels/keycard authentication.dm

```diff
@@ -150,6 +150,8 @@
 			revoke_maint_all_access()
 			feedback_inc("alert_keycard_auth_maintRevoke",1)
 		if("Emergency Response Team")
+			if(SSticker.mode.ert_disabled)
+				SSticker.mode.announce_ert_disabled()

```

Comment:
That should be thrown into the is_ert_blocked call below (or outright removed) as this is doing pretty much the same thing.
```
/obj/machinery/keycard_auth/proc/is_ert_blocked()
	if(config.ert_admin_call_only) return 1
	return SSticker.mode && SSticker.mode.ert_disabled
```